application/common/models/ResetBase.php

```diff
@@ -36,11 +37,13 @@ public static function tableName()
     public function rules()
     {
         return [
-            [['user_id', 'type', 'code', 'expires', 'created'], 'required'],
```

Comment:
yeah that is my guess too, maybe we should remove the default from the migration since it is implemented in the model too.